D/dav1d/pga.jl

```diff
@@ -0,0 +1,44 @@
+# Projective Geometric Algebra for General Relativity
```

Comment:
This doesn't look needed for this recipe.

Reply:
Weird. Thanks. I removed the file.